### PR TITLE
#4805 [Manager] Connector classes should not appear in Toolbox's stre…

### PR DESCRIPTION
**Pull request by konekoya**
…am section

### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #4805)
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [x] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/src/components/Pipeline/Toolbox/ToolboxHooks.js [hunk 1]
```diff
@@ -44,14 +44,20 @@ export const useTopics = () => {
...
+      if (classInfo.classType !== KIND.stream) return null;
+
+      return {
+        name: classInfo.className.split('.').pop(),
+        kind: KIND.stream,
+        className: classInfo.className,
+        jarKey: { name, group },
+      };
+    })
+    .filter(Boolean);
```
Comment (eechih) [inline ohara-manager/client/src/components/Pipeline/Toolbox/ToolboxHooks.js:60]:
```suggestion
  return classInfos
    .reduce((acc, cur) => {
      if (cur.classType === KIND.stream) {
        acc.push({ ... });
      };
      return acc;
    });
```

Comment (konekoya) [reply ohara-manager/client/src/components/Pipeline/Toolbox/ToolboxHooks.js:60]: Thanks @eechih ❤️

File: ohara-manager/client/src/components/Pipeline/Toolbox/ToolboxHooks.js [hunk 2]
```diff
@@ -42,27 +42,31 @@ export const useTopics = () => {
...
-  const streamFiles = hooks.useStreamFiles();
-  const streams = streamFiles
-    .map(restructureStream)
-    .filter(stream => !_.isEmpty(stream))
-    .sort((a, b) => a[0].className.localeCompare(b[0].className))
-    .reduce((acc, cur) => acc.concat(cur), []);
-
-  return streams;
+  return hooks
+    .useFiles()
```
Comment (konekoya) [inline ohara-manager/client/src/components/Pipeline/Toolbox/ToolboxHooks.js:67]: Now, all files are scanned and so not only stream jars' classInfo are extracted but all of them

Comment (konekoya) [reply ohara-manager/client/src/components/Pipeline/Toolbox/ToolboxHooks.js:67]: cc @eechih and @saivirtue

Comment (konekoya) [reply ohara-manager/client/src/components/Pipeline/Toolbox/ToolboxHooks.js:67]: I was thinking about moving the hook to `/hooks` but it seems the logic is very specific and is unlikely will be reused for other components. And so I kept it here :)

Comment (eechih) [reply ohara-manager/client/src/components/Pipeline/Toolbox/ToolboxHooks.js:67]: 看起來 Toolbox stream list 好像沒有參考 workspace.stream.jarKeys? 
需要加入到 Workspace 才能顯示在 Toolbox 上。

Comment (konekoya) [reply ohara-manager/client/src/components/Pipeline/Toolbox/ToolboxHooks.js:67]: Sorry @eechih, I'm not following @@~ So it's not working as expected?

Comment (eechih) [reply ohara-manager/client/src/components/Pipeline/Toolbox/ToolboxHooks.js:67]: Workspace settings 有一個 stream jars 的功能，上傳的 file 必須被挑選進入 stream jars，才能顯示在 Toolbox 上。

Comment (konekoya) [reply ohara-manager/client/src/components/Pipeline/Toolbox/ToolboxHooks.js]: Thanks for catching this. PR updated :)

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by konekoya** [comment]
retry manager-e2e

**Pull request merged**